src/app/shared/utils/testHelpers.tsx

```diff
@@ -5,7 +5,9 @@
 import * as React from 'react';
 import { shallow, mount } from 'enzyme';
 import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
```

Comment:
we should ignore this file from test coverage and remove the `testWithLocalizationContextAndErrorBoundary` as it is not being called by any tests.